### DISCUSSION TITLE
Custom Displays/Buttons/Etc.

### DISCUSSION THREAD

**Discussion by IIAIronWolf** [HASwitchPlate/HASPone#129] [Ideas]
Is there a good place to find other's custom versions of the HASP UI? I've seen some pretty cool looking versions of the UI online, but can't seem to find a good place to copy other's work. 
Maybe we should start a custom HASP community?

**Comment by aderusha** [comment]
This is the custom HASP community!  Previously everything was in one multi-thousand-comment long [thread on the Home Assistant community forums](https://community.home-assistant.io/t/ha-switchplate-haspone-diy-in-wall-touchscreen-home-assistant-controller/25464).  That was getting hard to navigate so I have started posting here where we have a forums-like UI that's a little closer to the HASPone project's code and wiki.

**Reply by IIAIronWolf** [reply]
I guess I was thinking, maybe we should add a space for custom UI's, like the Blueprint Exchange. Or is that what the exchange already is?

**Reply by aderusha** [reply]
That's exactly it, and several of the blueprints there were user submissions.  PR's are welcome!

**Reply by IIAIronWolf** [reply]
Awesome! I'm hoping to see more custom UI's like this: 
![image](https://user-images.githubusercontent.com/51146434/132270632-ccdd7294-d72d-4c07-a7c4-261e3bd47990.png)

**Comment by stampeder** [comment]
I realize that this is an old post but is there a location for the custom UI's? The above are quite nice.
